CMakeLists.txt

```diff
@@ -381,13 +374,27 @@ set_property(TARGET osrm-prepare PROPERTY INSTALL_RPATH_USE_LINK_PATH TRUE)
 set_property(TARGET osrm-datastore PROPERTY INSTALL_RPATH_USE_LINK_PATH TRUE)
 set_property(TARGET osrm-routed PROPERTY INSTALL_RPATH_USE_LINK_PATH TRUE)
 
-install(FILES ${InstallGlob} DESTINATION include/osrm)
+file(GLOB VariantGlob third_party/variant/*.hpp)
```

Comment:
Ha, while we're at breaking things: what is your take on ripping out third_party/mapbox_variant and replacing it with Boost's for the 5.0 release? One thing less to worry about (e.g. we ship a really outdated variant release from April 2015 --- no newer release available).